### DISCUSSION TITLE
Need RSS for GitHub Releases

### DISCUSSION THREAD

**Discussion by ericoporto** [community/community#21574] [New to GitHub]
<p>Hi,</p>
<p>I have a website that I was thinking to automate to pickup releases of a project and showing them.</p>
<p>I know GitHub used to have a way to get a release feed in RSS, but I can’t exactly figure it out.</p>
<p>Where can I find it?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi there :wave:</p>
<p>I believe this is what you were looking for.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      

      <a href="https://www.ronaldsvilcins.com/2020/03/26/rss-feeds-for-your-github-releases-tags-and-activity/" target="_blank" rel="noopener" title="08:44AM - 26 March 2020">ronaldsvilcins.com – 26 Mar 20</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://www.ronaldsvilcins.com/2020/03/26/rss-feeds-for-your-github-releases-tags-and-activity/" target="_blank" rel="noopener">RSS feeds for your Github releases, tags and activity</a></h3>

  <p>Find out about latest from projects you follow via RSS feed</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by ericoporto** [comment]
<p>Thanks, these will be very useful!</p>

**Comment by Luttik** [comment]
I would love to see a option here to filter just on real releases and skip all those with the pre-release tag.

**Reply by thierry1769** [reply]
Same here...
